chpl/chpl-service/src/main/java/gov/healthit/chpl/listener/QuestionableActivityListener.java

```diff
@@ -4,7 +4,6 @@
 
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
-import org.aspectj.lang.annotation.After;
```

Comment:
If the @After annotation went away, how does the Questionable Activity Listener get triggered?